dts/arm/nxp/nxp_lpc54xxx.dtsi

```diff
@@ -8,8 +8,19 @@
 #include <dt-bindings/clock/mcux_lpc_syscon_clock.h>
 #include <dt-bindings/gpio/gpio.h>
 #include <dt-bindings/i2c/i2c.h>
+#include <mem.h>
 
 / {
+	aliases{
+		gpio-0 = &gpio0;
+		gpio-1 = &gpio1;
+		mailbox-0 = &mailbox0;
```

Comment:
The commit message doesn't say why these are needed and I do not think that they are.

These redundant aliases used to be needed in the legacy devicetree macros that were deprecated in Zephyr 2.3, but we can refer to nodes directly by node label now. The node identifier `DT_NODELABEL(gpio0)` is just as good as `DT_ALIAS(gpio_0)`, so I'd rather not have any more legacy aliases added in.